package.json

```diff
@@ -30,6 +30,7 @@
     "lodash.throttle": "^4.1.1",
     "react": "^15.4.1",
     "react-native": "^0.42.0",
+    "react-native-actionsheet": "^2.0.5",
```

Comment:
BTW, have you looked at `react-native-action-sheet`  (which has an extra dash) It is by Exponent which are active and would expect the quality of the component and code support to be better long term.

We can probably replace it if needed in a next PR though. Let's get this one merged first.